templates/zerver/integrations/twitter.md

```diff
@@ -1,12 +1,26 @@
-See Twitter search results in Zulip! This is great for seeing and
+Fetch tweets from Twitter in Zulip! This is great for seeing and
```

Comment:
I feel that this formulation is a little redundant, but I can't think of anything better to make it clear that this is not only about search, but also about fetching a user's timeline.